OpenRA.Mods.Common/Projectiles/Missile.cs

```diff
@@ -559,7 +559,7 @@ int IncreaseAltitude(int predClfDist, int diffClfMslHgt, int relTarHorDist, int
 
 					|| (desiredVFacing == 0 // Upper part of incline surmounting manoeuvre
 						&& relTarHorDist <= loopRadius * WAngle.FromFacing(vFacing).Sin() / 1024
-							+ Exts.ISqrt(predAttHght * (2 * loopRadius - predAttHght)))); // Target too close to hit at current speed
+							+ Exts.ISqrt(predAttHght * (2 * loopRadius - predAttHght > 0 ? 2 * loopRadius - predAttHght : 0)))); // Target too close to hit at current speed
```

Comment:
perhaps
```suggestion
							+ Exts.ISqrt(Math.Abs(predAttHght * (2 * loopRadius - predAttHght))))); // Target too close to hit at current speed
```

Reply:
idk if we should use an absolute number here, also I don't know how to produce an usecase to test this function properly(loop radius and other parameters are too substract). All I know is, iosmeric map with height will have a tiny chance crash on it.

Reply:
as I mentioned we could use Math.Max(variable, 0) instead

Reply:
Tomorrow I will take a look at it. I think I should make a comment here about the future missile test.